XRTK-Core/Packages/com.xrtk.core/Editor/PackageInstaller.cs

```diff
@@ -186,5 +187,10 @@ public static void InstallConfiguration(MixedRealityPlatformServiceConfiguration
             AssetDatabase.SaveAssets();
             EditorApplication.delayCall += () => AssetDatabase.Refresh(ImportAssetOptions.ForceUpdate);
         }
+
+        public static void RegenerateGuids(string[] filePaths)
+        {
+            GuidRegenerator.RegenerateGuids(filePaths, false);
```

Comment:
Would this make more sense to be an overload on `GuidRegenerator` itself?

Reply:
AH I see, I made it internal so people wouldn't use it outside of the core project.